docs/admin.rst

```diff
@@ -333,7 +373,7 @@ Setting Submission Preferences
 By default, SecureDrop supports both text submissions and document uploads. If you
 only want to receive text messages, you can disable uploads as follows:
 
-* check the the **Prevent sources from uploading documents** checkbox
+* check the **Prevent sources from uploading documents** checkbox
```

Comment:
nice find, someone should make a linter that just catches `the the` repetitions ;-)

Reply:
:P